test/e2e/common/node/pod_resize.go

```diff
@@ -1012,6 +1013,565 @@ func doPodResizeErrorTests(f *framework.Framework) {
 	}
 }
 
+func doConsecutivePodResizeTests(f *framework.Framework) {
+	var podClient *e2epod.PodClient
+	ginkgo.BeforeEach(func() {
+		podClient = e2epod.NewPodClient(f)
+	})
+
+	type testCase struct {
+		isSynchronous    bool
+		name             string
+		containers       []e2epod.ResizableContainerInfo
+		containerPatches [][]v1.Container
```

Comment:
Similar to other tests, it might be better to use patch string. We don't want to wait for JSON marshaling between consecutive resize request (especially for asynchronous testcase)